### ISSUE TITLE
Accessibility !!!!!!!

### ISSUE THREAD

**Issue by ntoll** [invent-framework/invent#81]
A11y is important in an educational context. Furthermore, it should be *easy* to make one's Invent application accessible by default. We should reveal and make use of the W3C web standards for accessibility (see: https://developer.mozilla.org/en-US/docs/Web/Accessibility).
